reactos/lib/rtl/actctx.c

```diff
@@ -1407,8 +1407,8 @@ static BOOL parse_assembly_identity_elem(xmlbuf_t* xmlbuf, ACTIVATION_CONTEXT* a
         }
         else if (xmlstr_cmp(&attr_name, languageW))
         {
-            DPRINT1("Unsupported yet language attribute (%.*S)\n",
-                    attr_value.len, attr_value.ptr);
+            DPRINT("Unsupported yet language attribute (%.*S)\n",
+                   attr_value.len, attr_value.ptr);
```

Comment:
Yeah, i'm using the repo at git.reactos.org rather than github since it lags behind